examples/csharp/src/Twilio/Rest/FlexApi/V1/Credential/NewCredentialsResource.cs

```diff
@@ -126,11 +126,11 @@ public static NewCredentialsResource Create(
                                           string testString,
                                           int? testInteger = null,
                                           DateTime? testDate = null,
-                                          float? testNumberFloat = null,
+                                          float testNumberFloat = null,
```

Comment:
It seems like, testNumberFloat is mandatory since ? is removed.
But at the same time, we are keeping its default value as null.